lib/contracts/feeoraclev2/config.go

```diff
@@ -0,0 +1,66 @@
+package feeoraclev2
+
+import (
+	"sort"
+
+	"github.com/omni-network/omni/lib/evmchain"
+)
+
+const defaultBaseGasLimit = 100_000
+const defaultBaseDataBuffer = 100
+const defaultGasPerByte = 16
+
+const (
+	// Mainnets.
+	IDEthereum uint64 = 1
+
+	// Testnets.
+	IDSepolia uint64 = 11155111
+)
+
+type Config struct {
+	ID             uint64
+	BaseGasLimit   uint32
+	BaseDataBuffer uint32
+	GasPerByte     uint64
+}
```

Comment:
We should ascribe more meaning this name here. And maybe not expose it publicly. When thinking of exposed names think of import path. This would be `feeoraclev2.Config`. But this is data cost config, not fee oracle config.

Though most structs & functions we do not need to expose yet. Best to default to keeping them private to module until somethings needed

Reply:
Refactoring these to feeConfig and dataCostConfig. Making them private as they're only needed in the fee oracle for now.